sdk/python/flet/datatable.py

```diff
@@ -252,12 +253,12 @@ def cells(self, value):
 
```

Comment:
Check `DataTable` - all color properties are errored.

Reply:
Oh. Whats the error message?
Here is an example of what I used: `heading_row_color: Union[None, str, Dict[Union[MaterialState, MaterialStateString], str]]`
Any error?